src/spikeinterface/sorters/runsorter.py

```diff
@@ -593,7 +606,10 @@ def run_sorter_container(
 
     # clean useless files
     if delete_container_files:
-        os.remove(parent_folder / "in_container_recording.json")
+        if (parent_folder / "in_container_recording.json").exists():
+            os.remove(parent_folder / "in_container_recording.json")
```

Comment:
Good that this happens inside the container so we don't have to worry about windows immediate delete error.

Reply:
@h-mayorquin this is actually happening on the host machine! So I'm in favor of the additional protection :)